app/scripts/controllers/preferences-controller.ts

```diff
@@ -132,14 +136,16 @@ export type PreferencesControllerState = {
   knownMethodData: Record<string, string>;
   currentLocale: string;
   identities: Record<string, AccountIdentityEntry>;
-  lostIdentities: Record<string, object>;
+  lostIdentities: Record<string, AccountIdentityEntry>;
   forgottenPassword: boolean;
   preferences: Preferences;
   ipfsGateway: string;
   isIpfsGatewayEnabled: boolean;
   useAddressBarEnsResolution: boolean;
   ledgerTransportType: LedgerTransportTypes;
-  snapRegistryList: Record<string, object>;
+  // TODO: Replace `any` with type
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  snapRegistryList: Record<string, any>;
```

Comment:
Is an `any` necessary here? Looking at the mock object for `snapRegistryList` https://github.com/MetaMask/metamask-extension/blob/79feca9a0f422c924af314b01a72d4251b22d087/test/data/mock-state.json#L607, it looks like `Json` would be wide enough.

```suggestion
  // TODO: Replace with correct type
  snapRegistryList: Record<string, Json>;
```